File: app/src/main/java/com/kalashnyk/denys/airqualitymonitoring/main_operation/fragments/list_files_fragment/delete_all_items/DeleteAllFilesDialog.java
package com.kalashnyk.denys.airqualitymonitoring.main_operation.fragments.list_files_fragment.delete_all_items;

import android.os.Bundle;
import android.support.v4.app.DialogFragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.view.Window;

import com.kalashnyk.denys.airqualitymonitoring.R;
import com.kalashnyk.denys.airqualitymonitoring.main_operation.fragments.list_files_fragment.item.IDialogFileListener;

import butterknife.ButterKnife;
import butterknife.OnClick;

public class DeleteAllFilesDialog extends DialogFragment {

    private IDialogFileListener listener;

    public DeleteAllFilesDialog(IDialogFileListener listener) {
        this.listener = listener;
    }

    public static DeleteAllFilesDialog newInstance(IDialogFileListener listener) {
        DeleteAllFilesDialog dialog = new DeleteAllFilesDialog(listener);
        return dialog;
    }

    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View v = inflater.inflate(R.layout.dialog_delete_all_files, null);
        ButterKnife.bind(this, v);
        getDialog().getWindow().requestFeature(Window.FEATURE_NO_TITLE);
        return v;
    }

    @OnClick(R.id.btn_dialog_cancel_all_files)
    public void onDialogCancel() {
        this.dismiss();
    }

    @OnClick(R.id.btn_dialog_delete_all_files)
    public void onDialogDelete() {
        listener.deleteAllFiles();
        this.dismiss();
    }

}

File: app/src/main/java/com/kalashnyk/denys/main_operation/fragments/list_files_fragment/delete_all_items/DeleteAllFilesDialog.kt
package com.kalashnyk.denys.main_operation.fragments.list_files_fragment.delete_all_items

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.DialogFragment
import androidx.navigation.Navigation
import com.example.myapplication.R
import kotlinx.android.synthetic.main.fragment_delete_all_files_dialog.*
import kotlinx.android.synthetic.main.fragment_main.*


class DeleteAllFilesDialog : DialogFragment() {


    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_delete_all_files_dialog, container, false)
    }

//    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
//        super.onViewCreated(view, savedInstanceState)
//
//        btn_dialog_cancel_all_files.setOnClickListener {
//            val action = DeleteAllFilesDialogDirections.toMainFragmentAction()
//            Navigation.findNavController(it).navigate(action)
//        }
//
//        btn_dialog_delete_all_files.setOnClickListener {
//            val action = DeleteAllFilesDialogDirections.allItemsAction()
//            Navigation.findNavController(it).navigate(action)
//        }
//
//
//    }

}